vault/audit_test.go

```diff
@@ -406,12 +387,13 @@ func TestAuditBroker_LogRequest(t *testing.T) {
 		Request:  reqCopy,
 		OuterErr: reqErrs,
 	}
-	err = b.LogRequest(context.Background(), logInput, headersConf)
+	ctx := namespace.RootContext(context.Background())
```

Comment:
Is adding the `namespace.RootContext` wrapper around a background context here required, or just best practice?

Reply:
Required, we get a ErrNoNamespace otherwise.

Reply:
Due to merging of NoopAudit and now everyone gets a` n.formatter.AuditFormatWriter = &audit.JSONFormatWriter` I assume this became a requirement?

Reply:
Precisely.